src/Juvix/Core/HRAnn/Erasure.hs

```diff
@@ -1,21 +1,20 @@
 module Juvix.Core.HRAnn.Erasure where
 
+import Juvix.Library
 import qualified Juvix.Core.HR.Types as HR
 import Juvix.Core.HRAnn.Types
+import Juvix.Core.IR.TransformExt
+
+
+hrForgetter :: ExtTransformTE HRAnn HR.HR primTy primVal
+hrForgetter =
+  ExtTransformTE
```

Comment:
We should name the fields, since it's easy to mix up the order.

Reply:
Done, and also for `TransformExt.forgetter`.